src/test/java/io/pinecone/PineconeClientTest.java

```diff
@@ -0,0 +1,110 @@
+package io.pinecone;
+
+import io.pinecone.exceptions.PineconeConfigurationException;
+import io.pinecone.clients.Pinecone;
+import org.mockito.ArgumentCaptor;
+import org.openapitools.client.model.*;
+import okhttp3.*;
+import org.junit.jupiter.api.Test;
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.AbstractMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+public class PineconeClientTest {
```

Comment:
This covers testing aspects of the `Client` builder.

Reply:
Nice!